packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -335,10 +333,12 @@ export async function copyParameterJson(
   const targetParameterFilePath = path.join(parameterFolderPath, targetParameterFileName);
   const sourceParameterFilePath = path.join(parameterFolderPath, sourceParameterFileName);
   const targetParameterContent = await fs.readJson(sourceParameterFilePath);
-  const appName = ctx.projectSettings!.appName;
-  targetParameterContent["parameters"][resourceBaseName] = {
-    value: generateResourceBaseName(appName, targetEnvName),
-  };
+  if (targetParameterContent["parameters"][resourceBaseName]) {
```

Comment:
should we also make `"parameters"` as a const? the coding style is not aligned

Reply:
sure